core/src/main/java/org/frankframework/senders/ShadowSender.java

```diff
@@ -206,7 +206,7 @@ private Message collectResults(Map<ISender, ParallelSenderExecutor> executorMap,
 		}
 
 		builder.close();
-		return Message.asMessage(builder.toString());
+		return new Message(builder.toString());
```

Comment:
Ik denk dat het verstandig is om de `XmlBuilder` een `asMessage()` methode te geven. Dat kan het beste in een los pr, en dan moeten we ook gelijk even kijken naar de toString methode, die is denk ik niet handig zo.. misschien dat we daar beter `asString()` van kunnen maken. En dan overal `toXML()` te vervangen.